cowbird/api/webhooks/views.py

```diff
@@ -34,6 +35,7 @@ def dispatch(handler_fct):
             exceptions.append(exception)
             LOGGER.error("Exception raised while handling event [%s] for handler [%s] : [%r].",
                          event_name, handler.name, exception)
+            traceback.print_exc()
```

Comment:
I think this is done automatically if you pass `exc_info=exception` to `LOGGER.error`.